jumpscale/packages/admin/frontend/components/settings/Settings.vue

```diff
@@ -70,9 +70,17 @@
                 :label="`Allow over provisioning`"
                 @click.stop="setDeveloperOptions()"
               ></v-switch>
+              <v-btn
+                hide-details
+                class="my-2 pl-2"
```

Comment:
You can use `ml-2` instead of `pl-2` to align with the above.

Reply:
thank you I updated it